apps/console/src/extensions/configs/common.tsx

```diff
@@ -27,6 +27,9 @@ export const commonConfig: CommonConfig = {
     blockLoopBackCalls: false,
     checkForUIResourceScopes: false,
     enableOrganizationAssociations: false,
+    footer: {
+        customFooterClassName: ""
```

Comment:
do we really need `Footer` ?
```suggestion
        customClassName: ""
```

Reply:
Fixed with 9e80de9